### ISSUE TITLE
Kill notification when read

### ISSUE THREAD

**Issue by Spidored** [jlippold/WifiSMS#12]
I really have no idea on the way to do that,

But i could be great to kill the notification on the screen (and make it read) of the iphone when we have read the SMS on the browser.

Cause when we have a long discussion, the notification come, and come and come ... ... and come ... xD

We'll arrived to "+15" and 15 notification to "read"/kill on the screen. 

I don't know how to do that, but the issue is to "kill" or mark read the notification too (and not only the SMS).

**Comment by treason** [comment]
I agree but I am not sure how to do this yet. I have an idea on how to do this with iRealSMS but not system wide.

**Comment by Spidored** [comment]
i'll investigate too by my side. i'll try to learn programming for iPhone too for help u better than saying things :p
